src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -262,6 +323,77 @@ export default {
 
             this.time = this.timeFormat.format(datum);
             this.imageUrl = this.imageFormat.format(datum);
+        },
+        getFormatter(key) {
+            let valueFormatter = this.openmct.telemetry.getValueFormatter({
+                format: key
+            });
+
+            return valueFormatter.formatter;
+        },
+        numericImageAge() {
+            let currentTime = this.currentTimeValue();
+            let selectedImage = this.getSelectedImage();
+            let formatter = this.getFormatter(this.timeKey);
+
+            if (selectedImage === undefined || selectedImage[this.timeKey] === undefined) {
+                return;
+            }
+
+            let parsedSelectedTime = formatter.parse(selectedImage[this.timeKey]);
+
+            return currentTime - parsedSelectedTime;
+        },
+        formattedAge() {
+            let age = this.numericImageAge();
+
+            if (!age) {
+                return;
+            }
+
+            let result = this.durationFormatter.format(age);
+
+            if (age > EIGHT_HOURS) {
+                let negativeAge = (age / ONE_HOUR) * -1;
+                result = moment.duration(negativeAge, 'hours').humanize(true);
+
+                if (age > TWENTYFOUR_HOURS) {
+                    negativeAge = (age / TWENTYFOUR_HOURS) * -1;
+                    result = moment.duration(negativeAge, 'days').humanize(true);
+                }
+            }
+
+            return result;
+        },
+        trackAge() {
+            let age;
+            this.clearAgeTracking();
+            this.ageTracker = window.setInterval(() => {
+                age = this.formattedAge();
+                if (age) {
+                    this.age = age;
+                } else {
+                    this.age = 'N/A';
+                }
+            }, AGE_TRACK_INTERVAL_MS);
+        },
+        clearAgeTracking() {
+            window.clearInterval(this.ageTracker);
+        },
+        isImageNew(lessThanMinutes) {
```

Comment:
Yeah I had included that in case we wanted to change what determines what "new" is for images, BUT if we're going to be doing that we'll be editing the code anyway so I guess it's moot! I'll make it a computed.